scio-test/src/it/scala/com/spotify/scio/avro/TypedAvroIT.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2018 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.avro
+
+import com.google.common.io.Files
+import com.spotify.scio.{ContextAndArgs, ScioResult}
+import com.spotify.scio.testing.PipelineSpec
+
+class TypedAvroIT extends PipelineSpec {
```

Comment:
Don't really need to extend `PipelineSpec` right? Not even using any matchers so `extends FlatSpec` should be enough.

OTOH, is there anything we should assert after running the job?

Reply:
👍 can change to just stick with `FlatSpec`. I thought about asserting that the file contents are as expected but it seemed a bit out of scope for this test...happy to include something like that if you think it's worthwhile though. I mainly wrote the test to make sure the fix works.

Reply:
Maybe use the `noException should be thrownBy` matcher then?
http://www.scalatest.org/user_guide/using_matchers

Reply:
👍